docs/src/api/class-pageerror.md

```diff
@@ -0,0 +1,23 @@
+# class: PageError
+* since: v1.38
+
+[PageError] class represents objects created by context when there are unhandled
+execeptions thrown on the pages and dispatched via the [`event: BrowserContext.pageError`] event
+
+```js
```

Comment:
We should add here snippets for the other language ports before merging.

Reply:
Added https://github.com/microsoft/playwright/pull/24452/commits/636e710a01ff8e3982c372459fe96372dbfd0d06